src/status_im/ui/components/badge.cljs

```diff
@@ -9,9 +9,12 @@
                  {:height 18 :border-radius 9 :min-width 18 :padding-horizontal 6}
                  {:height 22 :border-radius 11 :min-width 22 :padding-horizontal 8})
                {:background-color colors/blue
-                :justify-content :center
-                :align-items :center})
-   [react/text {:style {:typography :caption :color colors/white}} label]])
+                :justify-content  :center
+                :align-items      :center})
+   [react/text {:style {:typography  :caption
+                        :font-weight "500"
```

Comment:
`font-weight` is set in typography, so no need to set it here

Reply:
Yes, you are right, but in typography `:caption` is merged with `default-style` which has `font-weight` 400. Because caption is used in other places, I decided to make change locally to not break other specs. Maybe I can create another typography style for that?

Reply:
@errorists wdyt?

Reply:
but if it should be by design to use :caption and "500" then its fine

Reply:
I think it's up to you guys, how you prefer to implement text styles. tbh these aren't gospel and on occasion we have to use local overrides that don't have a counterpart in the typography stylesheet. the result is correct as in `font-size: 12` and `font-weight: 500`, I certainly appreciate looking up correct values in Figma, thanks!